src/Microsoft.ML.DataView/CiphertextDataViewType.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Diagnostics;
+using System.Threading;
+using Microsoft.ML.Data;
+using Microsoft.Research.SEAL;
+
+namespace Microsoft.ML.DataView
+{
+    public class CiphertextDataViewType : StructuredDataViewType
```

Comment:
All public types and members need xml doc comments

Reply:
Done.